src/main/java/edu/oakland/helper/admin/TrackData.java

```diff
@@ -123,6 +123,20 @@ public float getSpeed() {
     return points;
   }
 
+  @Override
+  public String toString() {
+    StringBuilder str = new StringBuilder();
+    if (isValid()) {
+      System.out.println("Status: " + getStatusMessage());
+      System.out.println("Speed: " + getSpeed());
+      System.out.println("Course Direction: " + getCourseDirection());
```

Comment:
Make sure these get appended to the output!
```suggestion
      str.append("Speed: " + getSpeed() + "\n");
      str.append("Course Direction: " + getCourseDirection() + "\n");
```